### PR TITLE
4 get article id endpoint

### PR DESCRIPTION
**Pull request by RGBlife**

### CODE REVIEW

**Comment by two-triangles** [comment]
Good Morning Bret, it seems that you have some merge conflicts that need to be resolved. Could you address these and then resubmit your PR using the nchelp pr command 👍

File: __tests__/app.test.js [hunk 1]
```diff
@@ -57,3 +57,49 @@ describe("Error controllers", () => {
...
+  });
+
+  test("Returns a status and error message when given an invalid article id", async () => {
+    const {
+      body: { msg },
+    } = await request(app).get("/api/articles/JELLY").expect(400);
+    const expected = "Bad request";
+
+    expect(msg).toBe(expected);
+  });
```
Comment (Harrison365) [inline __tests__/app.test.js:104]: Nice sad path tests 👍

File: controllers/errors.controller.js [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+    }
+    };
+
+    exports.handlePsqlErrors = (err, req, res, next) => {
+    const { code } = err;
+    if (code === "22P02") {
+        return res.status(400).send({ msg: "Bad request" });
+    } else {
+        next(err);
+    }
```
Comment (Harrison365) [inline controllers/errors.controller.js:20]: Nice 👍

**Comment by Harrison365** [comment]
#4 Good to go ✅

**Pull request merged**